.github/workflows/macos-syscollector-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: Syscollector test on macOS
+
+on:
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  build:
+    runs-on: macos-11
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
```

Comment:
We could generate a [python-version-file](https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-file-input) and use the same version for all tests (current and future).